client/homebrew/pages/basePages/listPage/brewItem/brewItem.less

```diff
@@ -19,6 +19,21 @@
 	break-inside                : avoid;
 	box-shadow                  : 0px 4px 5px 0px #333;
 	background-color            : #cab2802e;
+	.thumbnail  {
+		width: 80px;
+		height: 100px;
+		border: 2px solid #58180D;
+		border-radius: 8px;
+		float: left;
+		margin-right: 5px;
+		display: flex;
+		img.thumbnailImage {
```

Comment:
There is some slight positioning weirdness where the image is not quite filling the whole box. I'm not sure why, but there must be some tweak to make that fit perfectly. 
![image](https://user-images.githubusercontent.com/5878534/188298231-4d6db1c2-e3bd-47e8-a7c6-1d4688fe8166.png)

Reply:
This can be fixed by removing the `border` and `border-radius` from the container div, and applying instead directly on the image (assuming `object-fit: cover;` is also being used)